Makefile

```diff
@@ -9,8 +9,7 @@ generate-client:
 
 .PHONY: generate-client-local
 generate-client-local:
-	yq e -i -o=json ".info.version=\"${METAL_API_VERSION}\"" metal-api.json
-	yq e -o=json '.info.version' metal-api.json
+	docker run --rm -v ${PWD}:/etc/metal-go -u $(shell id -u):$(shell id -g) --entrypoint /bin/sh mikefarah/yq:4 -c 'yq e -i -o=json ".info.version=\"${METAL_API_VERSION}\"" /etc/metal-go/metal-api.json && yq e -o=json ".info.version" /etc/metal-go/metal-api.json'
```

Comment:
`/etc` is kind of a weird mount for a go project, maybe just mount it to `/work` or something?